examples/dashboard_basic_crud.py

```diff
@@ -0,0 +1,108 @@
+#!/usr/bin/env python
+#
+# Simple example of dashboard creation, retrieval, updating, and deletion.
+#
+
+import os
+import sys
+sys.path.insert(0, os.path.join(os.path.dirname(os.path.realpath(sys.argv[0])), '..'))
+from sdcclient import SdMonitorClient
+import uuid
+
+#
+# Parse arguments
+#
+if len(sys.argv) != 2:
+    print('usage: %s <sysdig-token>' % sys.argv[0])
+    print('You can find your token at https://app.sysdigcloud.com/#/settings/user')
+    sys.exit(1)
+
+sdc_token = sys.argv[1]
+
+#
+# Instantiate the SDC client
+#
+sdclient = SdMonitorClient(sdc_token)
+
+#
+# Create Dashboard.
+#
+ok, res = sdclient.create_dashboard("Sample dashboard - " + uuid.uuid4().hex)
+
+#
+# Check for successful creation
+#
+if not ok:
+    print(res)
+    sys.exit(1)
+
+dashboard = res['dashboard']
+
+#
+# Get Dashboard.
+#
+ok, res = sdclient.get_dashboard(dashboard['id'])
+
+#
+# Check for successful retrieval
+#
+if not ok:
+    print(res)
+    sys.exit(1)
+
+dashboard = res['dashboard']
+
+#
+# Update Dashboard.
+#
+dashboard['scopeExpressionList'] = [{
+  "displayName": None,
+  "isVariable": False,
+  "operand": "host.hostName",
+  "operator": "equals",
+  "value": ["my-host"],
+  "variable": False
+}]
+ok, res = sdclient.update_dashboard(dashboard)
+# NOTE: ID is optional if present in data set. Line below would also work as expected.
+#       This can be leveraged to copy one dashboard to another.
+# ok, res = sdclient.update_dashboard(dashboard, dashboard['id'])
+
+#
+# Check for successful update
+#
+if not ok:
+    print(res)
+    sys.exit(1)
+
+dashboard = res['dashboard']
+
+#
+# Delete Dashboard.
+#
+ok, res = sdclient.delete_dashboard(dashboard)
+
+#
+# Check for successful delete
+#
+if not ok:
+    print(res)
+    sys.exit(1)
+
+#
```

Comment:
I think the "CRUD" operations have been properly shown already. The following section seems to overlap the example and be a duplicate.

Was there anything specific to show here?

Reply:
how to use an existing dashboard's data to copy over to other dashboards. Primarily for customers using the SDC client to copy dashboards from one installation to another, etc.

Reply:
Essentially to override an existing dashboard with the configuration of a different one?

It's a good use case, I'd probably just keep the example `dashboard basic CRUD` more to the point.

You could create a separate example file to address the specific use case, however it also sounds a little too specific. I wonder if it actually adds value to the set of examples, since it seems to address a very specific user need.

Also, note that the current example may not work: in order to override a dashboard, you must retain the "destination" `id` and `version` and override all the other properties. I haven't tested it specifically, I'll review it after the other changes.

Let me know how you'd like to proceed. Thanks!